web/src/components/storage/LvmPage.test.tsx

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { screen, within } from "@testing-library/react";
+import { installerRender, mockParams } from "~/test-utils";
+import { model, StorageDevice } from "~/types/storage";
+import { apiModel } from "~/api/storage/types";
+import { gib } from "./utils";
+import { Drive } from "~/types/storage/model";
+import LvmPage from "./LvmPage";
+
+const sda1: StorageDevice = {
+  sid: 69,
+  name: "/dev/sda1",
+  description: "Swap partition",
+  isDrive: false,
+  type: "partition",
+  size: gib(2),
+  shrinking: { unsupported: ["Resizing is not supported"] },
+  start: 1,
+};
+
+const sda: StorageDevice = {
+  sid: 59,
+  isDrive: true,
+  type: "disk",
+  vendor: "Micron",
+  model: "Micron 1100 SATA",
+  driver: ["ahci", "mmcblk"],
+  bus: "IDE",
+  busId: "",
+  transport: "usb",
+  dellBOSS: false,
+  sdCard: true,
+  active: true,
+  name: "/dev/sda",
+  size: 1024,
+  shrinking: { unsupported: ["Resizing is not supported"] },
+  systems: [],
+  partitionTable: {
+    type: "gpt",
+    partitions: [sda1],
+    unpartitionedSize: 0,
+    unusedSlots: [{ start: 3, size: gib(2) }],
+  },
+  udevIds: ["ata-Micron_1100_SATA_512GB_12563", "scsi-0ATA_Micron_1100_SATA_512GB"],
+  udevPaths: ["pci-0000:00-12", "pci-0000:00-12-ata"],
+  description: "",
+};
+
+const mockSdaDrive: Drive = {
```

Comment:
NP: I would use `model.Drive`. Otherwise I have to go back to the imports to know whether it means `apiModel.Drive` or `model.Drive`.

Reply:
I see. I didn't do it on purpose, but thinking about it I see more benefits importing it as it is now that name spacing. If I'm not wrong, If for some reason we end up with dead code in the file  holding "model", tree shaking will not be able to ignore it when making the build, since we are importing the whole moduleninstead of the needed parts. So, maybe it can be imported `as ModelDrive` or we could even choose only one of these module for importing it completely and the other in parts. I.e, whatever not prefixed comes always from the same module.

Of course, that approach is a bit harder to follow, and we’re already importing these modules fully in here and there.

Reply:
BTW, types/storage/model exports everything and then "model" and not "default". Definitely I'm fully lost with it comes to storage area and the way we do things there.